BHoMAnalytics_UI/CaptureProjectData.xaml.cs

```diff
@@ -43,57 +43,132 @@ namespace BH.UI.Analytics
     /// </summary>
     public partial class CaptureProjectData : Window
     {
-        public CaptureProjectData()
+        #region Constructor
+        public CaptureProjectData(string uiName)
         {
             InitializeComponent();
-            this.ShowDialog();
+            VersionTextBlock.Text = $"BHoM Version: {BH.Engine.Base.Query.BHoMVersion()}";
+            NonProjectListBox.ItemsSource = Enum.GetValues(typeof(NonProjectOption));
+            this._uiName = uiName;
+            
+
+            if (_uiName == "Grasshopper")
+            {
+                GrassHopperUIText.Visibility = Visibility.Visible;
+            }
+            if (_uiName == "Revit")
+            {
+                BHoMUseTextBlock.Text = "Please indicate purpose of this use of BHoM Tools for Revit";
+            }
+
             ProjectBtn.Focus();
+            this.ShowDialog();
         }
+        #endregion
+
+        #region Properties
+        public string Version;
+        public string _uiName;
+
+        public int _initialHeight = 240;
+        #endregion
 
+        #region EventActions
+        //Project
         private void Click_ProjectBtn(object sender, EventArgs e)
         {
+            ResetForms();
             ProjectInputPanel.Visibility = Visibility.Visible;
-            this.Height = 250;
+            this.Height = 380;
             ProjectIDInput.Focus();
         }
+        private void Click_ConfirmProjectBtn(object sender, EventArgs e)
+        {
+            ConfirmProject();
+        }
+        private void KeyDown_ProjectIDEntry(object sender, KeyEventArgs e)
+        {
+            if (e.Key == Key.Enter || e.Key == Key.Return)
+                ConfirmProject();
+        }
 
+        //Non Project
```

Comment:
Put this comment inside the method so that if the method later moves, the comment will go with it - though I don't think the comment is actually needed given the name of the method?